CDCgov/prime-reportstream
# 8241 Update email address in Terms of Service

This PR ...

Updates the email address in Terms of Service

Test Steps:
1. Go to RS
2. Click on Terms of Service in the footer
3. Verify the email address is reportstream@cdc.gov

## Changes
- changed from usds@cdc.gov to reportstream@cdc.gov

## Checklist

### Testing
